tests/auth_plugin_test.php

```diff
@@ -237,6 +237,25 @@ public function test_throwing_exception_if_user_is_not_exist() {
         $this->expectExceptionMessage('Invalid parameter value detected (User is not exist)');
         $actual = $this->auth->get_login_url($user);
     }
+    /**
+     * Test that auth plugin throws correct exception if we trying to request not existing user.
+     */
+    public function test_throwing_exception_if_user_is_suspended() {
+        global $USER, $SESSION;
+
+        $user = $this->getDataGenerator()->create_user();
+        $user->suspended = 1;
+        $this->setUser($user);
```

Comment:
What are you trying to test here? I don't think you need set a user here and then try to login again, but using a key.